e2e/tests/hmctsSuperUserAdministersCase_test.js

```diff
@@ -1,18 +1,22 @@
 const config = require('../config.js');
 
 const orderCaseData = require('../fixtures/testData/caseDataWithOrderCollection.json');
+const caseManagementCaseData = require('../fixtures/standardDirectionOrder.json');
```

Comment:
maybe the fixture could have a more appropriate name, it's not an SDO it's case data in prepare for hearing state (containing an SDO)

Reply:
Could update in my other PR (#1650)

Reply:
sure